daemon/policy.go

```diff
@@ -157,6 +157,9 @@ func (h *getPolicyResolve) Handle(params GetPolicyResolveParams) middleware.Resp
 type AddOptions struct {
 	// Replace if true indicates that existing rules with identical labels should be replaced
 	Replace bool
+	// CleanRulesWithLabels if present, deletes all labels with the following
+	// labels.
+	CleanRulesWithLabels labels.LabelArray
```

Comment:
See the comment above, IMO it would be better the new option be named `Replaces` and the old `Replace` options be deleted.

Reply:
@jrajahalme 

I've changed the AddOptions to:
```golang
// AddOptions are options which can be passed to PolicyAdd
type AddOptions struct {
	// Replace if true indicates that existing rules with identical labels should be replaced
	Replace bool
	// ReplaceWithLabels if present indicates that existing rules with the
	// given LabelArray should be deleted.
	ReplaceWithLabels labels.LabelArray
}
```

I can't remove the Replace as it is used in other places in the code.